scripts/test.sh

```diff
@@ -0,0 +1,4 @@
+./target/release/polymesh purge-chain --dev -y
+./target/release/polymesh --dev &>/dev/null &
+cd scripts/cli
+npm test
```

Comment:
It is good to add the kill switch for the chain as well once the `npm test` completed.

Reply:
It will automatically be killed when the script exits.